templates/corporate/jobs.html

```diff
@@ -1,5 +1,4 @@
-{% extends "zerver/portico.html" %}
-{% set entrypoint = "landing-page" %}
+{% extends "zerver/portico.html" %} {% set entrypoint = "landing-page" %}
```

Comment:
I'm not sure but it's possible line-wrapping this code will change its behavior; we should not do this.